core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -355,6 +358,7 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
       _ <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.create(project) }
 
       _ <- samDAO.createResource(SamResourceTypeNames.billingProject, billingProjectName.value, ownerUserInfo)
+      _ <- samDAO.createResourceFull(SamResourceTypeNames.googleProject, project.projectName.value, Map.empty, Set.empty, ownerUserInfo, Option(SamFullyQualifiedResourceId(project.projectName.value, SamResourceTypeNames.billingProject.value)))
```

Comment:
GitHub won't let me comment because there weren't changes near that line, but can you also add an attempted cleanup of the `google-project` resource like we do with the `billing-project` resource on line 377?